src/extra/AutoUI/Filters/PersistentFilters.tsx

```diff
@@ -148,11 +144,8 @@ export const PersistentFilters = ({
 	// and we were able to load some from the local storage, then communicate them
 	// back to the parent component.
 	React.useEffect(() => {
-		updateUrl(storedFilters);
-		if (!filters?.length && storedFilters.length && onFiltersUpdate) {
-			onFiltersUpdate(storedFilters);
-		}
-	}, []);
+		filtersUpdate(storedFilters);
```

Comment:
Does this need the `updateUrl` that `filtersUpdate()` triggers?
If yes, then let's plz move this useEffect bellow the definition of `filtersUpdate`.
If not (since `filtersUpdate()` is also called by `<Filters />`'s `onFiltersUpdate()`), I would say let's just call `onFiltersUpdate(filters);`.

Reply:
It is not needed, good catch. Replaced this call with just `onFiltersUpdate`,